src/yuzu/main.cpp

```diff
@@ -1208,64 +1125,82 @@ void GMainWindow::ConnectWidgetEvents() {
-    connect(ui->action_Reset_Window_Size_900, &QAction::triggered, this,
-            &GMainWindow::ResetWindowSize900);
-    connect(ui->action_Reset_Window_Size_1080, &QAction::triggered, this,
-            &GMainWindow::ResetWindowSize1080);
-    ui->menu_Reset_Window_Size->addAction(ui->action_Reset_Window_Size_720);
-    ui->menu_Reset_Window_Size->addAction(ui->action_Reset_Window_Size_900);
-    ui->menu_Reset_Window_Size->addAction(ui->action_Reset_Window_Size_1080);
+    connect_to(ui->action_Reset_Window_Size_720, &GMainWindow::ResetWindowSize720);
+    connect_to(ui->action_Reset_Window_Size_900, &GMainWindow::ResetWindowSize900);
+    connect_to(ui->action_Reset_Window_Size_1080, &GMainWindow::ResetWindowSize1080);
+    ui->menu_Reset_Window_Size->addActions({ui->action_Reset_Window_Size_720,
+                                            ui->action_Reset_Window_Size_900,
+                                            ui->action_Reset_Window_Size_1080});
 
     // Tools
-    connect(ui->action_Rederive, &QAction::triggered, this,
-            std::bind(&GMainWindow::OnReinitializeKeys, this, ReinitializeKeyBehavior::Warning));
-    connect(ui->action_Capture_Screenshot, &QAction::triggered, this,
-            &GMainWindow::OnCaptureScreenshot);
+    connect_to(ui->action_Rederive,
+               std::bind(&GMainWindow::OnReinitializeKeys, this, ReinitializeKeyBehavior::Warning));
+    connect_to(ui->action_Capture_Screenshot, &GMainWindow::OnCaptureScreenshot);
 
     // TAS
-    connect(ui->action_TAS_Start, &QAction::triggered, this, &GMainWindow::OnTasStartStop);
-    connect(ui->action_TAS_Record, &QAction::triggered, this, &GMainWindow::OnTasRecord);
-    connect(ui->action_TAS_Reset, &QAction::triggered, this, &GMainWindow::OnTasReset);
-    connect(ui->action_Configure_Tas, &QAction::triggered, this, &GMainWindow::OnConfigureTas);
+    connect_to(ui->action_TAS_Start, &GMainWindow::OnTasStartStop);
+    connect_to(ui->action_TAS_Record, &GMainWindow::OnTasRecord);
+    connect_to(ui->action_TAS_Reset, &GMainWindow::OnTasReset);
+    connect_to(ui->action_Configure_Tas, &GMainWindow::OnConfigureTas);
 
     // Help
-    connect(ui->action_Open_yuzu_Folder, &QAction::triggered, this, &GMainWindow::OnOpenYuzuFolder);
-    connect(ui->action_About, &QAction::triggered, this, &GMainWindow::OnAbout);
+    connect_to(ui->action_Open_yuzu_Folder, &GMainWindow::OnOpenYuzuFolder);
+    connect_to(ui->action_About, &GMainWindow::OnAbout);
+}
+
+void GMainWindow::UpdateMenuState() {
+    const bool is_paused = emu_thread == nullptr || !emu_thread->IsRunning();
+
+    const auto running_actions = {ui->action_Stop,
+                                  ui->action_Restart,
+                                  ui->action_Configure_Current_Game,
+                                  ui->action_Report_Compatibility,
+                                  ui->action_Load_Amiibo,
+                                  ui->action_Pause};
```

Comment:
and run clang-format
```suggestion
                                  ui->action_Pause,};
```

Reply:
clang-format doesn't do anything here.

Reply:
Given its usage elsewhere I'll add the trailing comma but this isn't part of clang-format. Is that style just not included?

Reply:
The whole yuzu codebase use a coma after the last element of arrays or enums. This allows for better clang formatting.
![image](https://user-images.githubusercontent.com/5944268/142952644-bcfc6a82-53f3-4f54-ad15-2165fb6f1397.png)

Reply:
Basically what narr said, I wouldn't have suggested this if it wasn't used everywhere for better formatting

Reply:
Yep realized after I wrote what I wrote. This should be part of some kind of checkstyle if possible.